v2/client.go

```diff
@@ -1015,3 +1015,8 @@ func (c *Client) NewManagedSubAccountAssetsService() *ManagedSubAccountAssetsSer
 func (c *Client) NewSubAccountFuturesAccountService() *SubAccountFuturesAccountService {
 	return &SubAccountFuturesAccountService{c: c}
 }
+
+// NewSubAccountFuturesSummaryV1Service Get Summary of Sub-account's Futures Account (For Master Account)
+func (c *Client) NewSubAccountFuturesSummaryV1Service() *SubAccountFuturesSummaryV1Service {
```

Comment:
```suggestion
func (c *Client) NewSubAccountFuturesSummaryService() *SubAccountFuturesSummaryService {
```

This implementation looks solid, however, I'd recommend reconsidering the naming convention used for NewSubAccountFuturesSummaryV1Service. Adding V1 to the name might not be the best approach for future-proofing. If binance introduce a v2 API, we could simply update the URL within the same function. This way, users won't have to modify their existing codebase with each new version of the service. It's a small change but can significantly improve the maintainability and usability of our API. What do you think?

Reply:
@adshao there is already v2 which is totally different from the v1. I wanted the v1, that's why the naming. 
V1: https://binance-docs.github.io/apidocs/spot/en/#get-summary-of-sub-account-39-s-futures-account-for-master-account
V2: https://binance-docs.github.io/apidocs/spot/en/#get-summary-of-sub-account-39-s-futures-account-v2-for-master-account